examples/common/avb.h

```diff
@@ -112,11 +112,11 @@ typedef enum { false = 0, true = 1 } bool;
 
 int pci_connect(device_t * igb_dev);
 
-int gptpscaling(gPtpTimeData * td, char *memory_offset_buffer);
+int gptpscaling(char *igb_mmap, gPtpTimeData *td);
```

Comment:
Just pointing out that the `gptpscaling()` arguments were reversed to match `jackd_talker` and `simple_talker`.  Did anything else use `gptpscaling()` from `common/avb` ?